src/utils/retry.ts

```diff
@@ -1,19 +1,27 @@
+import { Logger } from '@utils/logger';
+import { lightFormat } from 'date-fns';
+
 type AsyncOperation<T> = () => Promise<T>;
 interface RetryOptions {
   retries: number;
   delay: number;
 }
 export const retry = async <ReturnType, ErrorType extends Error = Error>(operation: AsyncOperation<ReturnType>, options: RetryOptions, errorsToStop: Array<new (...args: any[]) => ErrorType>): Promise<ReturnType> => {
   try {
+    Logger.info({ prefix: '<<<retry function>>>', message: `Retrying again... Retries left: ${options.retries}` });
```

Comment:
You may show this particular message when retrying and not the first time. It may be a bit confusing to see The retrying message in the logger screen.
![image](https://github.com/sirius-valley/tricker-backend/assets/104390122/8800bb62-0792-4653-817f-25f7343db251)